equinoxfitness/datacoco-cloud
# Initial upload to pypi

## Description

https://equinoxfitness.atlassian.net/browse/DATPLAT-1430


## Changes

- Add pull request template
- Add unit tests
- Add tox
- Prepare code to be uploaded to pypi


### How Has This Been Tested?

1. Checkout this branch

1. Switch to virtual environment

1. Run `tox`
- [ ] Verify that the test runs successfully


### Rollback Plan

What to do if something goes wrong?
